docs/projects/fps/tutorial.md

```diff
@@ -137,7 +141,7 @@ To find out about how to do this, read up about [worker attribute sets](https://
 
 Once an entity template function exists you have a way of constructing the _template_ of an entity. You now have a couple of ways of adding a health pack entity to the world:
 
-* At runtime, by passing an `Entity` object to an entity creation function.
+* At runtime, by passing an `EntityTemplate` object to an entity creation function.
```

Comment:
```suggestion
* At Runtime, by passing an `EntityTemplate` object to an entity creation function.
```

Reply:
This doesn't refer to the `SpatialOS Runtime` so should this be capitalized? This is just the general program runtime (or execution time)

Reply:
💥